Bool IsSynced
Bool SequensStarted
Bool MachineMoving
int8 SequenseNr
float X
float Y
float Z
!StartTime
!FinishedTime
!ErrorNr